tests/js/spec/components/eventsTable/__snapshots__/eventsTableRow.spec.jsx.snap

```diff
@@ -1,25 +1,134 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`EventsTableRow renders 1`] = `
-<tr
-  key="904"
->
-  <td>
-    <h5>
-      <Link
-        onlyActiveOnIndex={false}
-        style={Object {}}
-        to="/orgId/projectId/issues/groupId/events/904/"
-      >
-        <DateTime
-          date="2017-07-26T00:34:20Z"
-          seconds={true}
-        />
-      </Link>
-      <small>
-        TypeError: Cannot read property 'assignedTo' of undefined
-      </small>
-    </h5>
-  </td>
-</tr>
+<EventsTableRow
```

Comment:
Can we update to use the undecorated component?

Reply:
Oh yeah, didn't realise this has changed - updated